sql/views.sql

```diff
@@ -87,9 +87,20 @@ GROUP BY user_id,nutrient_id;
 \! echo " * macro_totals_today"
 # View: Food Totals
 CREATE VIEW total_food_logs AS
-SELECT user_id,food_id,nutrient_id,servings,date FROM food_log
-INNER JOIN macronutrient_content USING (food_id)
-GROUP BY user_id,food_id ORDER BY date ASC;
+select t.user_id,
+       f.name as 'Food',
+       t.servings as 'servings',
+       SUM(CASE WHEN n.id = 1 THEN mi.amount * t.servings ELSE '0' END) as 'fat',
+       SUM(CASE WHEN n.id = 4 THEN mi.amount * t.servings ELSE '0' END) as 'carbs',
+       SUM(CASE WHEN n.id = 7 THEN mi.amount * t.servings ELSE '0' END) as 'protein',
+       SUM(CASE WHEN n.id = 6 THEN mi.amount * t.servings ELSE '0' END) as 'fiber',
+       t.date as 'Date'
```

Comment:
This query is good and works with all macros. Servings and calories should also be `SUM()`;ed. `date` is not part of the group-by clause and cannot be summed, so it should be removed.